eventlet/green/ssl.py

```diff
@@ -260,6 +259,7 @@ def do_handshake(self):
             super(GreenSSLSocket, self).do_handshake)
 
     def _socket_connect(self, addr):
+        clock = hubs.get_hub().clock
```

Comment:
get clock should be in the else of if self.gettimeout() is None:
it is not used in other cases

Reply:
Thanks for caring of performance so much, fixed.